examples/audit_log_export/README.md

```diff
@@ -317,80 +316,6 @@ couchbase-capella_audit_log_export.new_auditlogexport: Modifying... [id=b173a42e
-
-Terraform will perform the following actions:
-
-  # couchbase-capella_audit_log_export.new_auditlogexport will be destroyed
-  - resource "couchbase-capella_audit_log_export" "new_auditlogexport" {
-      - audit_log_download_url = "https://cb-audit-logs-1234.s3.amazonaws.com/export/cluster-audit-logs-5678-from-2000-01-01-to-2000-01-02.tar.gz?X-Amz-Algorithm=X&X-Amz-Credential=XXX&X-Amz-Date=1999&X-Amz-Expires=1&X-Amz-Security-Token=XXX&X-Amz-SignedHeaders=host&X-Amz-Signature=XXX" -> null
-      - cluster_id            = "41938b91-66ed-4c84-b70c-55c3f0ae4266" -> null
-      - created_at            = "2024-03-13 06:48:43.596764031 +0000 UTC" -> null
-      - end                   = "2024-03-13T06:44:15+00:00" -> null
-      - expiration            = "2024-03-16 06:48:46.64587117 +0000 UTC" -> null
-      - id                    = "b173a42e-3c2c-4245-af81-f8615d63b5ce" -> null
-      - organization_id       = "637cea1d-fce5-40a5-9d48-8d0690e656ee" -> null
-      - project_id            = "6aa13dbf-69cb-48e3-97af-d89f57ea7f90" -> null
-      - start                 = "2024-03-13T04:44:15+00:00" -> null
-    }
-
-Plan: 0 to add, 0 to change, 1 to destroy.
-
-Changes to Outputs:
-  - existing_auditlogexport = {
-      - cluster_id      = "41938b91-66ed-4c84-b70c-55c3f0ae4266"
-      - data            = [
-          - {
-              - audit_log_download_url = "https://cb-audit-logs-1234.s3.amazonaws.com/export/cluster-audit-logs-5678-from-2000-01-01-to-2000-01-02.tar.gz?X-Amz-Algorithm=X&X-Amz-Credential=XXX&X-Amz-Date=1999&X-Amz-Expires=1&X-Amz-Security-Token=XXX&X-Amz-SignedHeaders=host&X-Amz-Signature=XXX"
-              - cluster_id            = "41938b91-66ed-4c84-b70c-55c3f0ae4266"
-              - created_at            = "2024-03-13 06:48:43.596764031 +0000 UTC"
-              - end                   = "2024-03-13 06:44:15 +0000 UTC"
-              - expiration            = "2024-03-16 06:48:46.64587117 +0000 UTC"
-              - id                    = "b173a42e-3c2c-4245-af81-f8615d63b5ce"
-              - organization_id       = "637cea1d-fce5-40a5-9d48-8d0690e656ee"
-              - project_id            = "6aa13dbf-69cb-48e3-97af-d89f57ea7f90"
-              - start                 = "2024-03-13 04:44:15 +0000 UTC"
-              - status                = "Completed"
-            },
-        ]
-      - organization_id = "637cea1d-fce5-40a5-9d48-8d0690e656ee"
-      - project_id      = "6aa13dbf-69cb-48e3-97af-d89f57ea7f90"
-    } -> null
-
-Do you really want to destroy all resources?
-  Terraform will destroy all your managed infrastructure, as shown above.
-  There is no undo. Only 'yes' will be accepted to confirm.
-
-  Enter a value: yes
-
-couchbase-capella_audit_log_export.new_auditlogexport: Destroying... [id=b173a42e-3c2c-4245-af81-f8615d63b5ce]
-╷
-│ Error: Audit Log Export does not support delete
-│
-│ Audit Log Export does not support delete
-```
-
 ## IMPORT
 ### Remove the resource from the Terraform State file
```

Comment:
I guess we do not need this command as of now. We should demonstrate the current delete behavior and highlight in the notes that it is a no-op operation, so it will only remove the resource from the state file.

### Remove the resource from the Terraform State file

Command:  `terraform state rm couchbase-capella_audit_log_export.new_auditlogexport`

Sample Output:
```
terraform state rm couchbase-capella_audit_log_export.new_auditlogexport
Removed couchbase-capella_audit_log_export.new_auditlogexport
Successfully removed 1 resource instance(s).

Reply:
I'd to keep the state removal command, as it can be useful to the user.  That said I agree example should be updated.

Addressed in a461b59f51e0eef3be49faade1e125bc649576f5